modules/ROOT/pages/dataweave-cookbook-use-regex.adoc

```diff
@@ -0,0 +1,75 @@
+= Use Regular Expressions in DataWeave
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, json, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping, as
+
+Several DataWeave functions accept regular expressions as arguments. You can use them to return or check for matches. You can also construct regular expressions that incorporate DataWeave expressions that include functions and variables.
```

Comment:
```suggestion
Several DataWeave functions accept regular expressions as arguments, which you can use to check for and return matches. You can also construct regular expressions that incorporate DataWeave expressions that include functions and variables.
```

Reply:
partially done. In some cases, you're just checking for whether there is a match, as in a Boolean (yes, no) vs. returning the matches.